Source/Mosa.DeviceSystem/Bitmap.cs

```diff
@@ -13,12 +15,14 @@ public static Image CreateImage(byte[] data)
 
 			var stream = new ByteStream(data);
 
-			//var bpp = stream.Read8(0x1C) / 8;
+			var sig = string.Empty + Encoding.ASCII.GetChar(stream.Read8(0)) + Encoding.ASCII.GetChar(stream.Read8(1));
```

Comment:
this creates a new string object, avoid creating throw away objects
this check could also be performed above alongside the bpp check
alternative check that doesn't perform any object allocations `(data[0x00] == (byte)'B' && data[0x01] == (byte)'M')`